cms/templates/js/no-group-configurations.underscore

```diff
@@ -1,3 +1,3 @@
 <div class="no-group-configurations-content">
-    <p><%= gettext("You haven't created any group configurations yet.") %><a href="#" class="button new-button"><i class="icon-plus"></i><%= gettext("Add your first Group Configuration") %></a></p>
+    <p><%= gettext("You haven't created any group configurations yet.") %><a href="#" class="button new-button"><i class="icon-plus"></i><%= gettext("Add your first Experiment Configuration") %></a></p>
```

Comment:
I'm not sure about our standard capitalization in other "Add your first..." type buttons. Typically I don't think object names need to have initial caps.